test/plugin_functional/plugins/core_plugin_appleave/public/plugin.tsx

```diff
@@ -15,6 +15,8 @@ export class CoreAppLeavePlugin
     core.application.register({
       id: 'appleave1',
       title: 'AppLeave 1',
+      appRoute: '/app/appleave1',
+      category: DEFAULT_APP_CATEGORIES.kibana,
```

Comment:
I cheated here a little, adding the test plugins to the Analytics section to find them without having to scroll through all the other plugins in the nav flyout.